controllers/fetchAllQuestions.js

```diff
@@ -0,0 +1,13 @@
+import express from 'express';
+import { data } from '../data/data';
+
+const allQuestions = data.questions;
+const fetchAllQuestionsCtrl = (req, res) => {
+  res.status(200);
+  res.json({
+  	status: 'successful',
+  	questions: allQuestions,
```

Comment:
You should stick to a consistent response body structure. It should always have a status, message and data attribute(where necessary). Implement this on all your response objects.